lib/active_record_shards/tasks.rb

```diff
@@ -12,7 +12,8 @@
       if key.starts_with?(env_name) && !key.ends_with?("_slave")
         begin
           if ActiveRecord::VERSION::MAJOR >= 4
-            ActiveRecord::Tasks::DatabaseTasks.drop(conf)
+            connection = ActiveRecord::Base.mysql2_connection(conf.merge('database' => nil))
```

Comment:
what's the difference between connection and mysql2_connection / shouldn't we be db agnostic ?